src/alchemy-apis/types.ts

```diff
@@ -151,6 +149,36 @@ export interface TransactionReceiptsResponse {
   receipts: TransactionReceipt[] | null;
 }
 
+export interface TransactionReceipt {
+  transactionHash: string;
+  blockHash: string;
+  blockNumber: string;
+  contractAddress: string | null;
```

Comment:
Can this be `contractAddress?` ?

Reply:
Adding `?` means that the value can also be `undefined`. `null` is treated differently and requires explicit typing.

Reply:
That's annoying - thanks for identifying this.